.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -167,7 +166,6 @@ jobs:
           TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
           TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
       - name: Post success to Slack channel dev-connectors-extensibility
-        if: success()
```

Comment:
Based on https://docs.github.com/en/actions/learn-github-actions/expressions#success, this is the default condition so no need to specify it here